domains/pallets/ethereum-beacon-client/src/config.rs

```diff
@@ -0,0 +1,39 @@
+#[cfg(feature = "minimal")]
+mod minimal;
+
+#[cfg(not(feature = "minimal"))]
+mod mainnet;
+
+#[cfg(feature = "minimal")]
+pub use minimal::*;
+
+#[cfg(not(feature = "minimal"))]
+pub use mainnet::*;
+
+pub const CURRENT_SYNC_COMMITTEE_INDEX: u64 = 22;
+pub const CURRENT_SYNC_COMMITTEE_DEPTH: u64 = 5;
+
+pub const NEXT_SYNC_COMMITTEE_DEPTH: u64 = 5;
+pub const NEXT_SYNC_COMMITTEE_INDEX: u64 = 23;
+
+pub const FINALIZED_ROOT_DEPTH: u64 = 6;
+pub const FINALIZED_ROOT_INDEX: u64 = 41;
+
+pub const MAX_PROPOSER_SLASHINGS: usize = 16;
+pub const MAX_ATTESTER_SLASHINGS: usize = 2;
+pub const MAX_ATTESTATIONS: usize = 128;
+pub const MAX_DEPOSITS: usize = 16;
+pub const MAX_VOLUNTARY_EXITS: usize = 16;
```

Comment:
would be great to add some comments explaining some of the constants here that are not self explanatory